packages/ds-healthcare-gov/src/styles/components/_ChoiceList.scss

```diff
@@ -0,0 +1,6 @@
+.ds-c-choice {
+  // healthcare is the only place we currently implement a border color for a choice error
+  &.ds-c-choice--error {
+    border-color: $choice__border-color--error;
+  }
```

Comment:
I was actually thinking that this would remain in core but without the condition, and then the value of `choice__border-color--error` would be set to the same value as `choice__border-color` for the themes where the feature would have been "off". I think we should minimize the amount of custom css in each theme.

Reply:
Ah ok sure I see.